src/EditorFeatures/CSharpTest/UseAutoProperty/UseAutoPropertyTests.cs

```diff
@@ -1403,8 +1383,89 @@ protected set
 }",
 @"class Class
 {
-
     public int P { get; protected set; }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)]
+        [WorkItem(26858, "https://github.com/dotnet/roslyn/issues/26858")]
+        public async Task TestPreserveTrailingTrivia1()
+        {
+            await TestInRegularAndScriptAsync(
+@"class Goo
+{
+    private readonly object [|bar|] = new object();
+
+    public object Bar => bar;
```

Comment:
>Bar [](start = 18, length = 3)

Not a new problem with this PR, but I don't see test scenarios where the property was defined above the field. Is that intentionally out-of-scope for the fixer?

Reply:
Have added tests.